docs/specs/v0.1.md

```diff
@@ -24,10 +24,13 @@
 \[`unsigned short` Green]
 \[`unsigned short` Blue]
 
+
 # Body
-\[`unsined short` FPS]
-\[`unsined short` Number of frames] # THIS IS A PROBLEM
-\[`unsined short` Number of people]
+\[`unsigned short` FPS]
+
+
+\[`unsigned short` Number of frames] 
```

Comment:
no need to modify the spec

Reply:
Should I put it back in?

Reply:
yeah i would revert the change to this file, no need to change it for this PR and there is just a comment removed

Reply:
Done!